.travis.yml

```diff
@@ -10,24 +12,42 @@ addons:
       - libgudev-1.0-dev
       - libxml++2.6-dev
 
-before_install:
-  - echo -n | openssl s_client -connect scan.coverity.com:443 | sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' | sudo tee -a /etc/ssl/certs/ca-
+stages:
```

Comment:
Travis apparently tries to run the Coverity stage prior to the default "test" stage unless you explicitly list `  - test` at the beginning of the stages list. No need to bother Coverity if the compile or unit tests fail.